webkit2/webkitdom.dom-object.lisp

```diff
@@ -0,0 +1,74 @@
+(in-package #:webkit2)
```

Comment:
File header missing.

Reply:
Still missing?

Reply:
Oh yes, I forgot about it! Should be fixed now :)